src/main/resources/mappings/missing-api-key-returns-401.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Returns a 401 for any url matching '/places/v1/addresses/postcode' with a 'postcode' query param. 
Requests that specify an api-key query param will be matched by future mappings.